composer.json

```diff
@@ -113,7 +115,8 @@
         "squizlabs/php_codesniffer": "^3.2",
         "escapestudios/symfony2-coding-standard": "^3.1",
         "friendsofphp/php-cs-fixer": "^2.10",
-        "ericmartel/codeception-email-mailtrap": "^1.0"
+        "ericmartel/codeception-email-mailtrap": "^1.0",
+        "tecnickcom/tcpdf": "6.*"
```

Comment:
It looks like this should be a regular requirement not a dev one.  Some deployments (like Heroku) will not install the dev requirements by default.